### PR TITLE
Feature add profile

### PR DESCRIPTION
**Pull request by jatnerubia**

### CODE REVIEW

**Review by Dev-Nino — APPROVED** [review]

File: src/app/add-profile/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ArrowLeftIcon from '@/_icons/arrow-left.icon'
+
+export const Header = () => {
+    return (
+        <div className="w-full flex justify-between px-6">
+            <ArrowLeftIcon />
+            <p className="font-semibold">Add Profile</p>
+            <span></span>
```
Comment (g-six) [inline src/app/add-profile/components/header.tsx:8]: What is this empth span for @jatnerubia ?

File: src/app/add-profile/components/template-form.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    const handleCreateProfile = () => {
+        setShowSuccess(true)
+    }
+
+    return (
+        <div className="w-full flex flex-col">
+            <div className="flex flex-col gap-[27px] px-6 pb-6">
+                <div className="flex flex-col">
+                    <label htmlFor="#" className="text-secondary-200 text-sm">
```
Comment (g-six) [inline src/app/add-profile/components/template-form.tsx:24]: `htmlFor="#"` looks like an unintentional mistake?

File: src/app/add-profile/components/template-light-form.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                            <React.Fragment key={index}>
+                                <div className="flex flex-col gap-4 px-6">
+                                    <p className="text-sm font-semibold">
+                                        Step {index + 1}
+                                    </p>
+                                    {/* Start time & duration */}
+                                    <div className="flex justify-between gap-2">
+                                        <div className="flex-1 flex flex-col gap-2">
+                                            <label
+                                                htmlFor=""
```
Comment (g-six) [inline src/app/add-profile/components/template-light-form.tsx:50]: remove this if not needed as blank value serves no purpose `htmlFor=""`

File: src/app/add-profile/components/template-light-form.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+                                                className="text-secondary-200 text-sm"
+                                            >
+                                                Start time
+                                            </label>
+                                            <TimeInput />
+                                        </div>
+                                        <div className="flex-1 flex flex-col gap-2">
+                                            <label
+                                                htmlFor=""
```
Comment (g-six) [inline src/app/add-profile/components/template-light-form.tsx:59]: remove this if not needed as blank value serves no purpose `htmlFor=""`

File: src/app/add-profile/components/template-light-form.tsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                                    <div className="flex items-center gap-3">
+                                        <p>0%</p>
+                                        <CustomSlider
+                                            value={100}
+                                            active={true}
+                                        />
+                                        <p>100%</p>
+                                    </div>
+                                    {/* Info */}
+                                    <div className="flex gap-1 bg-secondary-25 text-[10px] px-2 py-[6px] rounded-full">
```
Comment (g-six) [inline src/app/add-profile/components/template-light-form.tsx:103]: Please use propert tailwindcss sizes here.
`py-1.5` instead of py-[6px]

**Review by g-six — CHANGES_REQUESTED** [review]

File: src/app/add-profile/components/header.tsx (hunk 1 above)
Comment (jatnerubia) [reply src/app/add-profile/components/header.tsx:8]: need to center the `add profile`

**Comment by jatnerubia** [comment]
@g-six done ^^

**Review by g-six — APPROVED** [review]

**Pull request merged**